src/app/modules/did-book/services/did-book.service.ts

```diff
@@ -29,7 +30,11 @@ export class DidBookService {
   getList(): void {
     this.httpDidBook
       .getList()
-      .subscribe((list: DidBookRecord[]) => this.list.next(list));
+      .pipe(retryWhenWithDelay())
```

Comment:
Does `retryWhen` typically have an argument which defines when to retry? This is what I'm understanding from here: https://www.learnrxjs.io/learn-rxjs/operators/error_handling/retrywhen

Reply:
This is not retryWhen from rxjs. It is retryWhenWithDelay and is defined [here](https://github.com/energywebfoundation/switchboard-dapp/blob/develop/src/app/operators/retry-when-with-delay/retry-when-with-delay.ts)